src/js/base/api.js

```diff
@@ -325,6 +351,12 @@ ripe.Ripe.prototype._getMaskURL = function(options) {
     return options.url + "?" + this._buildQuery(options.params);
 };
 
+ripe.Ripe.prototype._getSwatchURL = function(material, color, options) {

```

Comment:
do we have this parameter standard anywhere else?

Reply:
We have for methods like `signin(username, password, options)` or `getOrder(number, options, callback)`

Reply:
But in terms of URL generation is there any that do receive more than the options?

Reply:
No, _getMaskURL and _getImageURL only receive options